src/lib/operators/operator_performance_data.hpp

```diff
@@ -26,6 +29,18 @@ struct OperatorPerformanceData : public Noncopyable {
                                 DescriptionMode description_mode = DescriptionMode::SingleLine) const;
 };
 
+struct StagedOperatorPerformanceData : public OperatorPerformanceData {
+  StagedOperatorPerformanceData() : OperatorPerformanceData{}, stage_runtimes{10} {}
```

Comment:
For some reason, something about "stage" doesn't feel right. The dictsch.. dick... wordbook tells me that it is a proper use, but personally, I would prefer "step". If you like stage better, that's fine. If you don't care, maybe we can change it?

Reply:
I am fine with steps as well. The only thing I can think about being slighly problematic with steps is that steps are one after another in my head. 
In several joins, the "steps" are running in parallel.

What about phase?

Reply:
Hmm ... then again, parallel steps doesn't sound bad to me.